Games/JV.AD/read.md

```diff
@@ -0,0 +1 @@
+This game is for students in grade 9 who want to learn grade 10 math. In this game, there will be questions relating to trigonometry, and the "SOHCAHTOA" formulas. These questions are not hard questions, just introductory questions meant to help students branch into grade 10 math more easily. This game is called "Big Trigs".
```

Comment:
Don't forget that you can use Markdown syntax to format the document.  Also the file name should be `README.md` for GitHub to automatically display this documentation when you browse to the folder.